scripts/release.cjs

```diff
@@ -38,7 +38,7 @@ module.exports = function release(pkgFolder, tag, skipBuild) {
   const newPkgTag = `${pkgName}@${newPkgVersion}`;
 
   console.log(execa.sync('git', ['add', '-A'], {cwd: dir}));
-  console.log(execa.sync('git', ['commit', '-m', `[skip ci] ${newPkgTag}`], {cwd: dir}));
+  console.log(execa.sync('git', ['commit', '--allow-empty', '-m', `[skip ci] ${newPkgTag}`], {cwd: dir}));
```

Comment:
Yeah I’m fine with this. Even though the commit is empty, the message is meaningful.

Reply:
Also a `try / catch` might bury important errors, so this WFM